src/components/OntologyBrowser/SelectionTree.css

```diff
@@ -36,3 +36,7 @@
 .ant-tree-node-content-wrapper .highlight {
   background-color: #f7ed9c;
 }
+
+.underline span {
```

Comment:
We need to validate with @luclemo but I think this should go in the main theme for all button of type link